Source/Blazorise/wwwroot/datePicker.js

```diff
@@ -200,6 +208,10 @@ export function updateOptions(element, elementId, options) {
             picker.set("altFormat", options.displayFormat.value);
         }
 
+        if (options.inputFormat.changed) {
+            setInputMask(picker, options.inputFormat.value);
```

Comment:
on line 94 I see : 
`setInputMask(picker, options.inputFormat);`

on this line I see:
`setInputMask(picker, options.inputFormat.value);`

Is there any diff?

Reply:
The `options.inputFormat.value` comes from component update, or when the parameter is changed. We do this convention all the time.